core/src/main/scala/filodb.core/store/IngestionConfig.scala

```diff
@@ -88,7 +86,7 @@ object StoreConfig {
 
   val defaults = ConfigFactory.parseString("""
                                            |disk-time-to-live = 3 days
-                                           |demand-paged-chunk-retention-period = 72 hours
+                                           |ingestion-time-index-retention = 7 days
```

Comment:
Is it not being used anywhere? Also, when default `disk-time-to-live` is 3 days(one line above), should this `ingestion-time-index-retention` be 7days here? Looks like something is missing here.

Reply:
Good point. I will revert default to 3 days.